dashboard/app/controllers/api_controller.rb

```diff
@@ -200,7 +202,16 @@ def section_assessments
               answer_indexes = Multi.find_by_id(level.id).correct_answer_indexes
               student_result = level_response["result"].split(",").sort.join(",")
               multi_count += 1
-              level_result[:student_result] = student_result
+
+              student_result_values = []
+              # Convert "0,1,3" to "A, B, D" for teacher-friendly viewing
+              student_result.split(',').each do |result|
+                student_result_values << multi_answer_characters[result.to_i]
+              end
+              viewable_student_result = student_result_values.join(', ')
+
+              level_result[:student_result] = viewable_student_result
```

Comment:
Is "A, B, D" more friendly or would it be better to display the actual answer text/image?  The letters also might be challenging if we decide to randomize the order of answers in the future (like we do for match).

Feel free to disregard, the letters are clearly better for now.